<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(194, 243, 237);</string>
  </property>
  <widget class="QLabel" name="songlabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>640</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <strikeout>false</strikeout>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>601</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget{
border:1px solid black;
border-radius:15px;
background-color:white;
}
QListWidget::hover{
background-color: rgb(178, 247, 236);
}
QListWidget::hover::doubleclicked{
background-color: rgb(194, 243, 237);
selection-color: white;
}
</string>
   </property>
  </widget>
  <widget class="QSlider" name="SongSlider">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>320</y>
     <width>480</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="repeatall">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>370</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>Repeat All</string>
   </property>
   <property name="toolTipDuration">
    <number>2000</number>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	border: 1px solid black;
	border-radius: 20px;
	background-color: white;
}
QPushButton::pressed
{ 
	background-color: rgb(138, 226, 52);
}</string>
   </property>
   <property name="text">
    <string>( A )</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="nextsong">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>370</y>
     <width>61</width>
     <height>40</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Next Song</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="previoussong">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>370</y>
     <width>61</width>
     <height>40</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Previous</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="repeatone">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>370</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:white;font-size:12pt; font-weight:600;&quot;&gt;Repeat One&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="toolTipDuration">
    <number>3000</number>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	border: 1px solid black;
	border-radius: 20px;
	background-color: white;
}
QPushButton::clicked
{
	background-color: rgb(138, 226, 52);
}


</string>
   </property>
   <property name="text">
    <string>( 1 )</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="removesong">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>39</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777210</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>Remove</string>
   </property>
   <property name="toolTipDuration">
    <number>3000</number>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red;
border:2px solid red;
border-radius : 10px;
background-color: white;</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addsong">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>Add</string>
   </property>
   <property name="toolTipDuration">
    <number>3000</number>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
color:green;
border:2px solid green;
border-radius : 10px;
background-color: white;
}
QPushButton::hover
{
	background-color: rgb(148, 252, 162);
}</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QLabel" name="songduration">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>320</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="songplayed">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="play_pause">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>360</y>
     <width>51</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>PLAY</string>
   </property>
   <property name="toolTipDuration">
    <number>2000</number>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid black;
border-radius : 25px;
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>I I</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QSlider" name="volumelevel">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>20</y>
     <width>16</width>
     <height>160</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="volume">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>190</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
border:1px;
border-radius: 3px ;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="musicaresource.qrc">
     <normaloff>:/icons/icons/volume.png</normaloff>:/icons/icons/volume.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="musicaresource.qrc"/>
 </resources>
 <connections/>
</ui>
